spec/acceptance/pe_status_check_spec.rb

```diff
@@ -284,6 +285,27 @@ class {'pe_status_check':
         expect(result.stdout).to match(%r{false})
         run_shell('touch -d "1 day ago"  /opt/puppetlabs/server/pe_build')
       end
+      context 'S0035' do
+        # verify pre-conditions are true
+        it 'returns true before adding an invalid module' do
+          result = run_shell('facter -p pe_status_check.S0035')
+          expect(result.stdout).to match(%r{true})
+        end
+
+        # verify false after action
+        it 'returns false after adding an invalid module' do
+          run_shell('puppet module install puppetlabs-ntp --version 9.1.0; puppet module install puppetlabs-stdlib --version 0.1.5 --force')
+          result = run_shell('facter -p pe_status_check.S0035')
+          expect(result.stdout).to match(%r{false})
+        end
+
+        # verify true after cleaning up
```

Comment:
we dont usually do the verify true after clean up, as that doesn't help with module acceptance, it would be nice but adds time to the test cycle